.env.template

```diff
@@ -1,4 +1,25 @@
-API_URI=https://demo.saleor.io/graphql/
 APP_MOUNT_URI=/
 APPS_MARKETPLACE_API_URI=https://apps.saleor.io/api/v2/saleor-apps
-LOCALE_CODE="EN"
\ No newline at end of file
+LOCALE_CODE="EN" 
+
+
+# automation tests vars
+DEMO_MODE=false
+# for legacy reasons CYPRESS vars are needed
+CYPRESS_USER_NAME=xxxx
+CYPRESS_USER_PASSWORD=xxxx
+CYPRESS_SECOND_USER_NAME=xxxx
+CYPRESS_PERMISSIONS_USERS_PASSWORD=xxxx
+CYPRESS_baseUrl=http://localhost:9000/
+
+
+E2E_USER_NAME=xxxx
+E2E_USER_PASSWORD=xxxx
+E2E_PERMISSIONS_USERS_PASSWORD=xxxx
+
+
+API_URI=https://demo.saleor.io/graphql/
```

Comment:
**suggestion**: I would keep that at the top. This is the most important variable